src/components/Company/Offers/Manage/CompanyOffersManagementWidget.spec.js

```diff
@@ -32,7 +35,15 @@ describe("App", () => {
         },
     ];
 
+    beforeEach(() => {
+        useSession.mockReturnValue({ data: { company: "company" }, isLoggedIn: true });
```

Comment:
The company id is missing. Do this instead:
```js
useSession.mockImplementation(() => ({
    isLoggedIn: true, data: { company: { name: "company1", _id: "company_id" } },
}));
```
Don't forget that the company id (in this case "company_id") should be equal to the `owner` property of the mocked offers.